app/src/main/java/org/thoughtcrime/securesms/conversation/start/NewConversationHomeFragment.kt

```diff
@@ -4,67 +4,76 @@ import android.os.Bundle
-        val adapter = ContactListAdapter(requireContext(), GlideApp.with(requireContext())) {
-            delegate.onContactSelected(it.address.serialize())
-        }
-        val unknownSectionTitle = getString(R.string.new_conversation_unknown_contacts_section_title)
-        val recipients = viewModel.recipients.value?.filter { !it.isGroupRecipient && it.address.serialize() != textSecurePreferences.getLocalNumber()!! } ?: emptyList()
-        val contactGroups = recipients.map {
-            val sessionId = it.address.serialize()
-            val contact = DatabaseComponent.get(requireContext()).sessionContactDatabase().getContactWithSessionID(sessionId)
-            val displayName = contact?.displayName(Contact.ContactContext.REGULAR) ?: sessionId
-            ContactListItem.Contact(it, displayName)
-        }.sortedBy { it.displayName }
-            .groupBy { if (PublicKeyValidation.isValid(it.displayName)) unknownSectionTitle else it.displayName.firstOrNull()?.uppercase() ?: unknownSectionTitle }
-            .toMutableMap()
-        contactGroups.remove(unknownSectionTitle)?.let { contactGroups.put(unknownSectionTitle, it) }
-        adapter.items = contactGroups.flatMap { entry -> listOf(ContactListItem.Header(entry.key)) + entry.value }
-        binding.contactsRecyclerView.adapter = adapter
-        val divider = ContextCompat.getDrawable(requireActivity(), R.drawable.conversation_menu_divider)!!.let {
-            DividerItemDecoration(requireActivity(), RecyclerView.VERTICAL).apply {
-                setDrawable(it)
+    @Composable
+    fun NewConversationScreen() {
+        Column(modifier = Modifier.background(MaterialTheme.colors.primarySurface)) {
+            AppBar(stringResource(R.string.dialog_new_conversation_title), onClose = { delegate.onDialogClosePressed() })
+            ItemButton(textId = R.string.messageNew, icon = R.drawable.ic_message) { delegate.onNewMessageSelected() }
+            Divider(startIndent = LocalDimensions.current.dividerIndent)
+            ItemButton(textId = R.string.activity_create_group_title, icon = R.drawable.ic_group) { delegate.onCreateGroupSelected() }
+            Divider(startIndent = LocalDimensions.current.dividerIndent)
+            ItemButton(textId = R.string.dialog_join_community_title, icon = R.drawable.ic_globe) { delegate.onJoinCommunitySelected() }
+            Divider(startIndent = LocalDimensions.current.dividerIndent)
+            ItemButton(textId = R.string.activity_settings_invite_button_title, icon = R.drawable.ic_invite_friend, contentDescription = R.string.AccessibilityId_invite_friend_button) { delegate.onInviteFriend() }
+            Column(
+                modifier = Modifier
+                    .padding(horizontal = LocalDimensions.current.marginMedium)
+                    .padding(top = LocalDimensions.current.itemSpacingMedium)) {
+                Text(text = stringResource(R.string.accountIdYours), style = MaterialTheme.typography.medium)
```

Comment:
This is set to XL on figma THOUGH I would check with the designer because the buttons above are also set to XL, which seems strange, as they are h8 in the rest of the app ( like the settings menu items )